bankCardPaymentValidator/app/dwolla-card-validator.js

```diff
@@ -0,0 +1,108 @@
+const https   = require('https');
+const ethers  = require('ethers');
+const fs      = require('fs');
+
+const root                = 'iexec_out';
+const determinismFilePath = `${root}/determinism.iexec`;
+const callbackFilePath    = `${root}/callback.iexec`;
+const errorFilePath       = `${root}/error.iexec`;
+
+
+/*****************************************************************************
+ *                                   TOOLS                                   *
+ *****************************************************************************/
+
+const Client = require("dwolla-v2").Client;
+
+/*****************************************************************************
+ *                                  CONFIG                                   *
+ *****************************************************************************/
+
+// dwolla api key
+const APIKEY = process.env.DWOLLA_APP_KEY;
+const SECRET = process.env.DWOLLA_APP_SECRET
```

Comment:
Where are these envvar comming from? I don't see them set anywhere. Should they come from a dataset? You cannot expect the worker to have them "by defaul"